### ISSUE TITLE
[Dreamcast] Depositing equipped frame with units, then equipping new frame and units causes equip slot already in use error

### ISSUE THREAD

**Issue by Matt-Swift** [fuzziqersoftware/newserv#207]
Steps to reproduce:
1. Equip any frame and any unit
2. Deposit equipped frame into bank
3. Equip new frame
4. Equip any unit (same slot presumably)

Log of issue: 
[Units.txt](https://github.com/fuzziqersoftware/newserv/files/13404713/Units.txt)

**Comment by nolrinale** [comment]
Can confirm it also affects BB

```
I 47528 2023-11-19 23:15:08 - [Commands] Received from C-6 (RUPIKA) (version=BB command=0060 flag=00000000)
0000 | 14 00 60 00 00 00 00 00 25 03 00 00 10 00 01 00 |   `     %       
0010 | 00 00 00 00                                     |                 
I 47528 2023-11-19 23:15:08 - [C-6] Equipped item 00010010
I 47528 2023-11-19 23:15:17 - [Commands] Received from C-6 (RUPIKA) (version=BB command=0060 flag=00000000)
0000 | 14 00 60 00 00 00 00 00 25 03 00 00 04 00 01 00 |   `     %       
0010 | 09 00 00 00                                     |                 
W 47528 2023-11-19 23:15:17 - [Server] Error processing client command: equip slot is already in use
I 47528 2023-11-19 23:15:17 - [Server] Client disconnected: C-6 on fd 37
I 47528 2023-11-19 23:15:17 - [Lobby:15] Deleted lobby
I 47528 2023-11-19 23:15:17 - [C-6] Deleted
I 47528 2023-11-19 23:15:17 - Added 44 seconds to play time
I 47528 2023-11-19 23:15:17 - Saved character file system/players/player_test_3.psochar

```

**Comment by fuzziqersoftware** [comment]
This is fixed by b03ccc7e.
